documents/Libraries/stdlib/stdlib_defs.mtlx

```diff
@@ -839,93 +698,67 @@
   <nodedef name="ND_noise3d_float" node="noise3d" type="float" nodegroup="procedural3d" default="0.0">
     <parameter name="amplitude" type="float" value="1.0"/>
     <parameter name="pivot" type="float" value="0.0"/>
-    <input name="position" type="vector3">
-      <geomprop name="position" type="vector3"/>
-    </input>
+    <input name="position" type="vector3" defaultgeomprop="Pobject"/>
```

Comment:
Same question about world vs object space. i.e. does texture "stick" to the object or does the object move through the texture. I thought it would be the latter.

Reply:
See above. (we can ask ILM if we're not sure what the best default behavior is)